.travis.yml

```diff
@@ -38,7 +38,8 @@ before_install:
     - sudo apt-get update -qq
 
     # Mongo C Driver
-    - git clone -b r1.2 https://github.com/mongodb/mongo-c-driver.git
+    # TODO switch master (branch for current 1.3 dev) to r1.3 when released
```

Comment:
Can you do this in a separate commit? Should definitely do it, but it isn't part of this ticket. Feel free to file a new ticket against 0.3.0 and send out a new PR. This had been on my TODO list.